### DISCUSSION TITLE
How is computed the accuracy for Suggest Features in circular projection ?

### DISCUSSION THREAD

**Discussion by Thibescobar** [biolab/orange3#5286] [Q&A] [answered]
Hello, 
The function **Suggest Features** in **Linear Projection** is very interesting. Can you explain how the combination of features are ranked please ? In the doc, it is mentioned that **"This feature scores attributes by average classification accuracy and returns the top scoring attributes with a simultaneous visualization update"**, but the way the accuracy is computed is not clear ([https://orange3.readthedocs.io/projects/orange-visual-programming/en/latest/widgets/visualize/linearprojection.html](https://orange3.readthedocs.io/projects/orange-visual-programming/en/latest/widgets/visualize/linearprojection.html)). Is it univariate accuracy of independent feature averaged across the combination of the selected features (for example 3 in the example of the doc)? Or is it a multivariate accuracy of a model made with theses features ? It would be very usefull to know since this tool is very usefull in my field (radiomics). Is the accuracy coputed by class or on the whole dataset ?
Thank you very much in advance.

**Comment by ajdapretnar** [comment]
@janezd @markotoplak

**Comment by janezd** [comment, marked as answer]
It computes the classification accuracy (or MSE, for regression) of k-nearest neighbours classifier on the projected (that is: two-dimensional) data. This measures how well the classes in projection are separated.

The number of projections may be huge, so they are evaluated in such order that combinations that include variables with higher ReliefF scores are tried first.

The basic idea is explained in http://eprints.fri.uni-lj.si/210/2/1._G._Leban%2C_B._Zupan%2C_G._Vidmar%2C_I._Bratko%2C_Data_Mining_and_Knowledge_Discovery_13%2C_119-36_(2006)..pdf.

**Reply by ajdapretnar** [reply]
I will add this to the documentation. Thanks!
